src/components/ManaCalculator/hooks/useManaState.ts

```diff
@@ -162,7 +171,27 @@ export function useGivenManaState(
   }
 
   function handleOwnHoldChange(value: number) {
-    setState({ ...state, heldTokens: value });
+    setState({
+      ...state,
+      ...getDerivedValuesFromHeldTokens(value),
+    });
+  }
+
+  function getDerivedValuesFromHeldTokens(
+    heldTokens: number,
+  ): Partial<ManaCalculatorProps> {
+    return {
+      heldTokens,
+      [getStakedOrDelegated(state.userType)]: heldTokens,
+      ...(state.userType === UserType.VALIDATOR
+        ? {
+            validator: {
+              ...state.validator,
+              attractedNewDelegatedStake: heldTokens,
```

Comment:
`attractedNewDelegatedStake` is not `heldTokens`, there is a formula https://github.com/iotaledger/iota-wiki/blob/main/src/components/ManaCalculator/hooks/useManaState.ts#L235, it doesnt depend on held tokens but it depends on stakedTokens and the validators set